pydra_ml/tasks.py

```diff
@@ -20,8 +23,9 @@ def read_file(filename, x_indices=None, target_vars=None, group="groups"):
     if group in data.keys():
         groups = data[:, [group]]
     else:
-        groups = list(range(X.shape[0]))
-    return X.values, Y.values, groups
+        groups = np.array(range(X.shape[0]))
```

Comment:
why does this need to be a numpy array?

Reply:
It doesn't, I think I was just testing if it changed anything when I was trying to figure out how to extract feature_names